cmdutil/debug/config.go

```diff
@@ -2,5 +2,8 @@ package debug
 
 // Config describes the configurable parameters for debugging.
 type Config struct {
-	Port int `env:"DEBUG_PORT,default=9999"`
+	Port                 int  `env:"DEBUG_PORT,default=9999"`
+	PprofPort            int  `env:"PPROF_DEBUG_PORT,default=9998"`
```

Comment:
Can we standardize on the naming of these environment variables? I'm thinking they should all follow the naming convention:
```
DEBUG_PPROF_XXX
```
so you would have:
```
DEBUG_PPROF_ENABLE
DEBUG_PPROF_MUTEX_PROFILE_ENABLE
DBEUG_PPROF_MUTEX_PROFILE_FRACTION
```